cypress/e2e/homepage.cy.js

```diff
@@ -15,10 +15,44 @@ describe('Test cases for W&Co', () => {
     cy.url().should('eq', 'https://work.co/clients/');
   })
   
-  it('Select Expertise and Capabilities',()=>{
+  it('Select Practise Areas',()=>{
     cy.get('[data-test-id="global-menu-btn"]').click()
     cy.get('[data-test-id="grid-practice-areas-link"]').click()
     cy.url().should('eq', 'https://work.co/company/');
   })
+  it('Go back to homepage',()=>{
+    cy.get('[data-test-id="global-menu-btn"]').click() 
+  })
+  it('Subscribe to Newsletter',()=>{
+    cy.get('[data-test-id="footer-subscribe-newsletter-link"]').click()
+    cy.get('[data-test-id="newsletter-subscribe-submit"]').should('be.disabled')
+    cy.get('[data-test-id="newsletter-email-textfield"]').click().type('zivanovic@work.co')
+    cy.get('[data-test-id="newsletter-subscribe-submit"]').should('be.enabled')
+
+
+  })
+  it('Check Disclaimer',()=>{
+    cy.get('.Subscribe-gdprDisclaimer').should('contain','We only send emails when we have something to say.')
+    cy.get('span').contains('Privacy Policy').parent().should('have.attr','href','https://mailchimp.com/legal/privacy/')
+    cy.get('span').contains('Terms').parent().should('have.attr','href','https://mailchimp.com/legal/terms/')
+    cy.get('[data-test-id="global-menu-btn"]').click()
+
+
+  })
+
+  it('Choose Belgarde lead position',()=>{
+    cy.get('[data-test-id="grid-careers-link"]').click()
+    cy.get('[data-test-id="grid-item-technology-link"]').click()
+    cy.get('[data-test-id="technology-grid-subitem-link-lead-developer-belgrade"]').click()
```

Comment:
Looks ok, but have in mind test will fail when they remove "Lead Developer" position from the available positions :D so for more consistent test you might use the "submit your application"  (which leads to https://work.co/careers/operations/global-studio/general-application/ ) since that general application is always available.
If you want some more practice, you can try uploading a file into "Select your resume/cv or drag it here" :)